go.mod

```diff
@@ -37,6 +37,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require golang.org/x/sys v0.30.0
+
```

Comment:
Nit: this entry belongs in the above list.

Reply:
Fixed in https://github.com/databricks/cli/pull/2376